moonraker/components/zeroconf.py

```diff
@@ -32,29 +40,65 @@ async def unregister_services(self, infos: List[AsyncServiceInfo]) -> None:
         await asyncio.gather(*background_tasks)
         await self.aiozc.async_close()
 
-class ZeroconfRegistrar():
+    async def update_services(self, infos: List[AsyncServiceInfo]) -> None:
+        assert self.aiozc is not None
+        tasks = [self.aiozc.async_update_service(info) for info in infos]
+        background_tasks = await asyncio.gather(*tasks)
+        await asyncio.gather(*background_tasks)
+        await self.aiozc.async_close()
```

Comment:
Should `async_close()` be called here?  I would think that this would shut down the zeroconf.

Reply:
Nope, missing commit.. one moment.